packages/subgraph/src/entities/Global.ts

```diff
@@ -154,10 +154,12 @@ export function handleLQTYStakeChange(
   } else if (stakeChange.stakeOperation == "stakeIncreased") {
     global.totalLQTYTokensStaked = global.totalLQTYTokensStaked.plus(stakeChange.amountChange);
   } else if (stakeChange.stakeOperation == "stakeDecreased") {
-    global.totalLQTYTokensStaked = global.totalLQTYTokensStaked.minus(stakeChange.amountChange);
+    // amountChange is a negative number
+    global.totalLQTYTokensStaked = global.totalLQTYTokensStaked.plus(stakeChange.amountChange);
   } else if (stakeChange.stakeOperation == "stakeRemoved") {
     global.numberOfActiveLQTYStakes--;
-    global.totalLQTYTokensStaked = global.totalLQTYTokensStaked.minus(stakeChange.amountChange);
+    // amountChange is a negative number
+    global.totalLQTYTokensStaked = global.totalLQTYTokensStaked.plus(stakeChange.amountChange);
```

Comment:
Now all 4 branches contain the same statement, which is
```typescript
global.totalLQTYTokensStaked = global.totalLQTYTokensStaked.plus(stakeChange.amountChange);
```
I think it makes sense to extract it from the `if`